doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -225,6 +225,12 @@ Asset Tracker v2
 
 * Removed the nRF7002 EK devicetree overlay file :file:`nrf91xxdk_with_nrf7002ek.overlay`, because UART1 is disabled through the shield configuration.
 
+Connectivity Bridge
+-------------------
+
+* Updated the :file:`README.txt` file of handling of emulated to support multiple boards.
```

Comment:
Wording seems a bit off here. Is this what you intend to write?

Updated :file:`README.txt` to support multiple boards

Reply:
I had suggested this change with the understanding that the readme file is of the handling of emulated. The original content was:  _Reworked handling of emulated README.txt to support multiple boards._

May be : _Update the handling of emulated in the :file:`README.txt` file  to support multiple boards._ 
Please confirm my understanding.

Reply:
fixed it